doc/getting_started/build/runtime_dependencies.md

```diff
@@ -12,7 +12,7 @@ When DLite is compiled with Python bindings, additional runtime features may be
 - [pint], used for units conversion in property mappings
 - [pydantic], used for testing support for pydantic models
 - [typing_extensions], needed by pydantic v2
-- [rdflib], used by `rdf` storage plugin
+- [rdflib], used by `pyrdf` storage plugin.  Is also used as default tripper backend for property mappings
```

Comment:
Why do I need to know about the default tripper backend at this point in the DLite documentation? Is it conventional to state dependencies of dependencies, or better to leave that description to the `tripper` documentation itself?

Reply:
Godt poeng. Vi kan ta bort andre setningen.